gateway/sds_gateway/users/forms.py

```diff
@@ -115,12 +186,61 @@ def clean_name(self):
 
         return name.strip()
 
-    def clean_author(self):
-        """Validate the author name."""
-        author = self.cleaned_data["author"]
-        if len(author.strip()) < MIN_AUTHOR_NAME_LENGTH:
-            raise ValidationError(AUTHOR_NAME_LENGTH_ERROR)
-        return author.strip()
+    def clean_authors(self):  # noqa: C901
```

Comment:
we can generate a couple of tests for this, to serve as regression for the current and old author object format

Reply:
yeah, that makes sense

Reply:
yeah, that makes sense